services/nomad/build/buildsync-aarch64.nomad

```diff
@@ -13,20 +14,11 @@ job "buildsync-aarch64" {
     }
 
     task "rsync" {
-      leader = true
       driver = "docker"
 
-      vault {
-        policies = ["void-secrets-buildsync"]
-      }
-
       config {
-        image = "eeacms/rsync"
-        args = ["client"]
-      }
-
-      env {
-        CRON_TASK_1="*/5 * * * * flock -n /run/rsync.lock rsync -vurk -e 'ssh -i /secrets/id_rsa -o UserKnownHostsFile=/local/known_hosts' --exclude '*.sig' --delete-after -f '+ */' -f '+ aarch64*-repodata' -f '+ *.aarch64*.xbps' -f '+ *.noarch*.xbps' -f '+ otime' -f '- *' void-buildsync@b-fsn-de.node.consul:/hostdir/binpkgs/ /pkgs/aarch64"
+        image = "ghcr.io/void-linux/infra-lsyncd:202308" # TODO
```

Comment:
Local is already bind mounted.  Doing a direct file mount means inode shift will result in file ghosting as the underlying template re-renders.  Consider either a symlink in the container for this file or a direct pointer as the startup invocation.